### ISSUE TITLE
Inconsistent icon name for video-camera icon

### ISSUE THREAD

**Issue by JackNeto** [themesberg/flowbite-svelte-icons#8]
Rename `video-solid` to `video-camera-solid`

https://github.com/themesberg/flowbite-svelte-icons/blob/main/src/lib/icons.js#L492

**Comment by shinokada** [comment]
Please see the [original icon name](https://github.com/themesberg/flowbite-icons/tree/main/src/solid/media).
The original icon name is video.

**Comment by JackNeto** [comment]
Let me rephrase that. 
On https://github.com/themesberg/flowbite-svelte-icons/blob/main/src/lib/icons.js the name of the outline and solid `video` icons are inconsistent with the other icons.  
Currently I can see `video-camera-outline` and `video-solid`. As you pointed out the name of the icon is `video` so the names in that file should be `video-outline` and `video-solid`

**Comment by shinokada** [comment]
Thank you for point it out. The [outline icon](https://github.com/themesberg/flowbite-icons/tree/main/src/outline/media) has the name with `video-camera`.
I use a script to auto-generate the files. 
Please create a issue at https://github.com/themesberg/flowbite-icons/issues?
